### ISSUE TITLE
Latest commit doesn't install on MacOS on M1

### ISSUE THREAD

**Issue by augustocdias** [neovim/neovim#19145]
### Neovim version (nvim -v)

0.8 01fc5097d3b6921fce5c159e5c772dc1590eb847

### Vim (not Nvim) behaves the same?

no

### Operating system/version

macOS 12.4

### Terminal name/version

iTerm

### $TERM environment variable

xterm-256color

### Installation

homebrew

### How to reproduce the issue

```
brew install neovim --HEAD
```

### Expected behavior

Successful instalation

### Actual behavior

```
HEAD is now at 01fc5097d Merge #19139 build(cmake): fix static libintl on macOS
==> Reinstalling neovim
==> luarocks make --lua-dir=/opt/homebrew/opt/luajit-openresty --tree=/private/tmp/neovim-20220629-36619-1uoyeqc/deps-build
==> luarocks make --lua-dir=/opt/homebrew/opt/luajit-openresty --tree=/private/tmp/neovim-20220629-36619-1uoyeqc/deps-build
==> make install PREFIX=/private/tmp/neovim-20220629-36619-1uoyeqc/deps-build LDFLAGS=-static
==> cmake -S . -B build -DLIBLUV_LIBRARY=/opt/homebrew/opt/luv/lib/libluv.dylib
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory @ rb_sysopen - build/config/auto/versiondef.h
```

**Comment by clason** [comment]
That looks like a homebrew issue @carlocab

**Comment by clason** [comment]
I cannot reproduce this. Are you building `luv` from HEAD as well? Otherwise it's probably a stale cache. Try `brew remove neovim`, `brew cleanup -s`, then `brew install neovim --HEAD` again.

**Comment by carlocab** [comment]
Fixed yesterday in Homebrew/homebrew-core#104638. Do `brew update` and try again.
